bdk-ffi/src/error.rs

```diff
@@ -0,0 +1,35 @@
+use crate::bitcoin::OutPoint;
+
+use bdk::chain::tx_graph::CalculateFeeError as BdkCalculateFeeError;
+
+use std::fmt;
+
+#[derive(Debug)]
+pub enum CalculateFeeError {
+    MissingTxOut { out_points: Vec<OutPoint> },
+    NegativeFee { fee: i64 },
+}
+
+impl fmt::Display for CalculateFeeError {
```

Comment:
`fmt::Display` implementation for `CalculateFeeError`: 
- `write!` is used to format and append the error message to the `fmt::Formatter`.
- this formatted message is used wherever the display representation of the error is needed